### DISCUSSION TITLE
Clean temporary directory

### DISCUSSION THREAD

**Discussion by Wulfheart** [livewire/livewire#1219] [4. General]
Is the temporary temporary_file_upload directory cleaned after a time n or does this have to happen manually?

**Comment by nuernbergerA** [comment]
On each upload, it will automatically delete all temp files older than 24 hours.

**Reply by logan-christensen** [reply]
Is there a way to configure clean up to happen more frequently?

**Reply by Wulfheart** [reply]
I don’t think so. The last time I checked there was no such configuration option.
